src/test/lrsql/util/reaction_test.clj

```diff
@@ -90,3 +90,68 @@
            (-> simple-reaction-ruleset
                r/serialize-ruleset
                r/deserialize-ruleset)))))
+
+
+(def fake-cond-map
+  {:completed_a {"id"     "6fbd600f-d17c-4c74-801a-2ec2e53231f7"
+                  "actor"  {"mbox" "mailto:bob@example.com"
+                            "madeUpAttr" "8"}
+                  "verb"   {"id" "https://example.com/verbs/completed"}
+                  "object" {"id"         "https://example.com/activities/a"
+                            "objectType" "Activity"}
+                  "result" [{"success" true} {"success" false}]}
+   :completed_b   {"id"     "c51d1628-ae4a-449f-8d8d-13d57207f468"
+                    "actor"  {"mbox" "mailto:bob@example.com"}
+                    "verb"   {"id" "https://example.com/verbs/completed"}
+                    "object" {"id"         "https://example.com/activities/b"
+                              "objectType" "Activity"}
+                    "result" [{"success" false} {"success" true}]}})
```

Comment:
Formatting is a little odd on these maps. If you have the Emacs `clojure-align` command, that will nicely format them for you with vertical alignment. Catch me on slack if you have trouble getting that set up.